sql/core/src/main/scala/org/apache/spark/sql/execution/stat/StatFunctions.scala

```diff
@@ -254,7 +254,8 @@ object StatFunctions extends Logging {
         stats.toLowerCase(Locale.ROOT) match {
           case "count" => (child: Expression) => Count(child).toAggregateExpression()
           case "mean" => (child: Expression) => Average(child).toAggregateExpression()
-          case "stddev" => (child: Expression) => StddevSamp(child).toAggregateExpression()
+          case "stddev" => (child: Expression) =>
+            StddevSamp("stddev_samp", child).toAggregateExpression()
```

Comment:
can we create a `object StddevSamp` and define the default name in the `apply` method?